rest/model/pod.go

```diff
@@ -41,6 +41,9 @@ type APICreatePod struct {
 
 // fromAPIEnvVars converts a slice of APIPodEnvVars to a map of environment variables and a map of secrets.
 func fromAPIEnvVars(api []*APIPodEnvVar) (envVars map[string]string, secrets map[string]string) {
```

Comment:
Can you make this a method on `(*APICreatePod)` since it's only used for the one purpose?

Reply:
Does that mean adding `(p *APICreatePod)` before the function name?

Reply:
Yes. Adding a receiver makes it a method on that particular type.